### PR TITLE
Moved minimum power filter into process.

### PR DESCRIPTION
**Pull request by ryan-rushton**
Cleaned up the usage of stat filters in the post process filter, that should be handled in process now.

### CODE REVIEW

File: src/app/loadout-builder/generated-sets/utils.ts [hunk 1]
```diff
@@ -211,35 +189,16 @@ export function filterGeneratedSets(
     return true;
   });
 
-  matchedSets = matchedSets.sort(
```
Comment (bhollis) [inline src/app/loadout-builder/generated-sets/utils.ts:214]: Where did sorting move to?

Comment (ryan-rushton) [reply src/app/loadout-builder/generated-sets/utils.ts:214]: Process is keeping them in a sorted order with the insertion sort. We could leave it in to be sure if you like.

Comment (ryan-rushton) [reply src/app/loadout-builder/generated-sets/utils.ts:214]: I pushed it back in. Still not entirely sure if its needed.

Comment (bhollis) [reply src/app/loadout-builder/generated-sets/utils.ts:214]: I think sorting by the stat order is probably important.

Comment (ryan-rushton) [reply src/app/loadout-builder/generated-sets/utils.ts:214]: Yeah it definitely is, I just think it is getting done twice (once in process and once here), just slightly differently.

**Pull request merged**